include/nuttx/syslog/syslog.h

```diff
@@ -84,12 +84,17 @@
 
 /* This structure provides the interface to a SYSLOG device */
 
-typedef CODE ssize_t (*syslog_write_t)(FAR const char *buf, size_t buflen);
-typedef CODE int (*syslog_putc_t)(int ch);
-typedef CODE int (*syslog_flush_t)(void);
+typedef CODE ssize_t (*syslog_write_t)(FAR void *dev, FAR const char *buf,
+                                       size_t buflen);
+typedef CODE int (*syslog_putc_t)(FAR void *dev, int ch);
+typedef CODE int (*syslog_flush_t)(FAR void *dev);
 
 struct syslog_channel_s
 {
+  /* Channel device */
+
+  void *sc_dev;
```

Comment:
```suggestion
  FAR void *sc_dev;
```
Missing FAR qualifier on common code.

Reply:
I'm sorry, I always forget `FAR`s... Added now.

Reply:
No problem!